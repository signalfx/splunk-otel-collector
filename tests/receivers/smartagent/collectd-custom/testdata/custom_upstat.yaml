receivers:
  smartagent/collectd/custom:
    type: collectd/custom
    template: |
      LoadPlugin "python"
      TypesDB "/var/collectd-python/upstat/upstat_types.db"
      <Plugin python>
        ModulePath "/var/collectd-python/upstat"
        Import "collectd-upstat"
        <Module "collectd-upstat">
        </Module>
      </Plugin>

exporters:
  otlp:
    endpoint: "${OTLP_ENDPOINT}"
    tls:
      insecure: true

service:
  telemetry:
    logs:
      level: "debug"
  pipelines:
    metrics:
      receivers:
        - smartagent/collectd/custom
      exporters: [otlp]
