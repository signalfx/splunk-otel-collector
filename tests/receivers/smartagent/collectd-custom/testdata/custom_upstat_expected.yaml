resourceMetrics:
  - resource: {}
    scopeMetrics:
      - metrics:
          - gauge:
              dataPoints:
                - asInt: "1"
                  attributes:
                    - key: dsname
                      value:
                        stringValue: value
                    - key: host
                      value:
                        stringValue: ef7bda3969ca
                    - key: plugin
                      value:
                        stringValue: upstat
                    - key: plugin_instance
                      value:
                        stringValue: upstat
                    - key: system.type
                      value:
                        stringValue: custom
                  timeUnixNano: "1000000"
            name: up
        scope: {}
